cmd/utils/flags.go

```diff
@@ -504,6 +504,10 @@ var (
 		Name:  "pruneancient",
 		Usage: "Prune ancient data, recommends to the user who don't care about the ancient data. Note that once be turned on, the ancient data will not be recovered again",
 	}
+	FDLimitFlag = cli.IntFlag{
+		Name:  "fdlimit",
+		Usage: "Raise the open file descriptor resource limit (default = system fd limit)",
```

Comment:
I am not sure whether this may cause chaos, because we register BSC node as a service and do set max open file limit there, need some test on this.